logstash/examples/security/test/goss.yaml

```diff
@@ -17,7 +17,8 @@ user:
     gid: 1000
 
 http:
-  http://localhost:9600?pretty:
+  https://localhost:9600?pretty:
+    allow-insecure: true
```

Comment:
This is Logstash endpoint. It should stay in HTTP and don't need `allow-insecure`